modules/thermal_hydraulics/src/parser/THMSyntax.C

```diff
@@ -24,7 +24,8 @@ associateSyntax(Syntax & syntax)
   syntax.registerActionSyntax("AddComponentAction", "Components/*", "THM:add_component");
   syntax.registerActionSyntax("AddComponentAction", "Components/*/*", "THM:add_component");
   syntax.registerActionSyntax("AddClosuresAction", "Closures/*", "THM:add_closures");
-  syntax.registerActionSyntax("THMAddControlAction", "ControlLogic/*", "add_control");
+  syntax.registerActionSyntax("THMAddControlAction", "ControlLogic/*", "THM:add_control");
+  syntax.registerTaskName("THM:add_control", "THMControl", false);
```

Comment:
Would it be better to to change `THM::add_control` to `add_control_logic`? @joshuahansel

Reply:
It's a little messy. The base class is `THMControl` even though the syntax/system is `ControlLogic`. The other tasks we've registered in THM are preceded with `THM::`. `THM::add_control_logic` is probably best, but I think either way is fine.

Reply:
changed